docs/source/version_compatibility.rst

```diff
@@ -0,0 +1,34 @@
+**************
+Sabre Versions
+**************
+
+The following table explains what versions of the Sawtooth Sabre SDK are
+compatible with different versions of the Sawtooth Sabre transaction
+processors.
+
+The Sabre SDK versions are the Crate versions of the Rust library that should
```

Comment:
Good info - thanks for adding it!  But ... please put the Sabre transaction processor first (to match the order of the columns in the table).

I recommend using bullets, one for each item:

* The Sabre transaction processor versions are ...
* The Sabre SDK versions are ...
* The Docker tag is ....

Also, Docker should be initial-capped (according to their doc and website).